### PR TITLE
core(metrics): move first-interactive to first-cpu-idle

### PR DESCRIPTION
**Pull request by patrickhulce**
1. Renames `first-interactive` to `first-cpu-idle` according to the new metric name
2. Moves core `first-interactive` computed artifact to a metric computed artifact

This is blocked on #4981 since that PR moves away from using first CPU idle for load-fast-4-pwa

### CODE REVIEW

File: lighthouse-core/test/results/sample_v2.json [hunk 1]
```diff
@@ -105,24 +105,8 @@
...
-              "url": "http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js",
-              "latency": "564.12"
-            }
-          ],
-          "isFast": true,
-          "timeToFirstInteractive": 4927.278
-        }
-      },
+      "rawValue": 4927.278,
+      "debugString": "First Interactive was 4,930 ms. More details in the \"Performance\" section.",
```
[the remark this answers is not in the document]
Comment (patrickhulce) [reply lighthouse-core/test/results/sample_v2.json]: yeah yeah need to update the JSON it was based on my branch before I updated the json there
